src/pages/components/TitleBar.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { IoIosArrowForward } from 'react-icons/io'
+import styled from 'styled-components'
+import { Box } from 'atomic-layout'
+
+const StyledTitleBar = styled.div`
+  background-color: ${({ theme }) => theme.colors.greyDim};
+  background-image: url('https://demo2.wpopal.com/auros/wp-content/uploads/2018/10/breadcrumb.jpg');
```

Comment:
Maybe consider hosting this image yourself? Just to be safe if that Auros resource is not available, so your website is not broken.

Reply:
I will do it in the future if you think necessary. But now all data depend on auros data.